addons/product/views/product_views.xml

```diff
@@ -592,50 +592,41 @@ action = {
         <field name="model">product.product</field>
         <field name="arch" type="xml">
             <kanban records_draggable="0" js_class="product_kanban_catalog">
-                <field name="id" invisible="1"/>
-                <field name="default_code" invisible="1"/>
-                <field name="image_128" invisible="1"/>
+                <field name="id"/>
                 <templates>
                     <t t-name="kanban-menu">
                         <div class="o_product_catalog_cancel_global_click">
-                            <a role="menuitem" type="edit" class="dropdown-item border-top-0">Edit</a>
+                            <a role="menuitem" type="open" class="dropdown-item border-top-0">Edit</a>
                         </div>
                     </t>
-                    <t t-name="kanban-box">
-                        <div class="d-flex flex-grow-1">
-                            <div class="oe_kanban_details p-2 d-flex">
-                                <div class="o_kanban_record_top flex-column m-0"
-                                     t-attf-id="product-{{record.id.raw_value}}">
-                                    <div class="d-flex">
-                                        <field style="margin-top: 2px;"
-                                            class="me-1"
-                                            name="is_favorite"
-                                            widget="boolean_favorite"
-                                            nolabel="1"
-                                            readonly="1"/>
-                                        <h4 class="text-reset">
-                                            <strong class="o_kanban_record_title"><field name="name"/></strong>
-                                        </h4>
-                                    </div>
-                                    <div t-if="record.default_code.value">
-                                        [<field name="default_code"/>]
-                                    </div>
-                                    <!-- Used by @web/product/js/product_catalog/order_line to
-                                         show the price using a t-portal. -->
-                                    <div name="o_kanban_price"
-                                         t-attf-id="product-{{record.id.raw_value}}-price"
-                                         class="d-flex flex-column"/>
-                                    <field name="product_template_attribute_value_ids"
-                                           widget="many2many_tags"
-                                           domain="[('id', 'in', parent.ids)]"
-                                           groups="product.group_product_variant"
-                                           options="{'color_field': 'color'}"/>
+                    <t t-name="kanban-card" class="p-0">
+                    <!-- TODO : not getting border when select record -->
```

Comment:
can i get your suggestion here how can i achieve this? :thinking:

Reply:
@aab-odoo ? o_product_added is added and the style applies but some other rule interferes.

Reply:
@omra-odoo in the product_catalog kanban view (in their kanban_record.xml, which should btw define an `<article>` node instead of a `<div role="article"/>`), you can set the o_product_added classname on the article. Then, in the associated scss (order_line.scss, wtf), change the selector
```scss
.o_kanban_view .o_kanban_renderer .o_kanban_record .o_product_added {
```
into
```scss
.o_kanban_view .o_kanban_renderer .o_kanban_record.o_product_added {
```

Reply:
It's working great, thank you for the suggestions.